.gitlab-ci.yml

```diff
@@ -10,7 +10,7 @@ variables:
 
   TF_TFU_VERSION: '1.0.0'
   TF_TFW_VERSION: '1.0.0'
-  TOUCHFREE_VERSION: '2.0.0'
+  TOUCHFREE_VERSION: '2.1.0'
```

Comment:
Should this be 2.0.0 as it's merging to develop before the release branch gets merged?